extensions-core/parquet-extensions/src/main/java/org/apache/druid/data/input/parquet/avro/ParquetAvroHadoopInputRowParser.java

```diff
@@ -92,6 +93,7 @@ private LogicalType determineTimestampSpecLogicalType(Schema schema, String time
   /**
    * imitate avro extension {@link org.apache.druid.data.input.avro.AvroParsers#parseGenericRecord}
    */
+  @Nonnull
```

Comment:
We aren't super consistent about this in the codebase but we strive to be better, I think this isn't necessary because we all assume `@Nonnull` is the default. You can enforce this with a package-info.java if you'd like, but it should be fine to leave out as well.

Reply:
Thanks, what is the path forward here? For me, explicit is better than implicit. WDYT

Reply:
I'm happy as well to make this more consistent in the codebase. Maybe we can also use SpotBugs to enforce this a bit.

Reply:
Ah, I see now, thanks for pointing this out. I think the setting everything to NonNull by default is the best practice, I'll pick this up in another PR if that is fine by you.